### PR TITLE
updated package-json to move all dev dependencies 

### PR DESCRIPTION
**Pull request by kulisfunk**
and installed all selenium test scripts. 2 dummy tests pass

### CODE REVIEW

File: client/src/models/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var express = require('express');
+var path = require('path');
+var router = new express.Router();
+
+
+
+router.use('/bucketlist', require('./bucketlist'));
+
```
Comment (Findsay) [inline client/src/models/index.js]: Don't think this should say bucketlist? Also this file should be under controllers?

**Review by Findsay — CHANGES_REQUESTED** [review]
The index.js file should be in controllers and has wrong router. Should be reverted back?

File: client/src/models/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+var express = require('express');
+var path = require('path');
+var router = new express.Router();
+
+
```
Comment (JamieSK) [inline client/src/models/index.js]: Are these blank lines here for something?

**Review by JamieSK — APPROVED** [review]
Looks good to me if they run fine

File: client/src/models/index.js (hunk 1 above)
Comment (JamieSK) [reply client/src/models/index.js]: I'm going to work on my proof reading

File: client/src/models/index.js (hunk 2 above)
Comment (kulisfunk) [reply client/src/models/index.js]: they were already there I just moved the file

File: client/src/models/index.js (hunk 1 above)
Comment (kulisfunk) [reply client/src/models/index.js]: what file are those in?

Comment (JamieSK) [reply client/src/models/index.js]: image

File: client/src/models/index.js (hunk 2 above)
Comment (JamieSK) [reply client/src/models/index.js]: Fair enough, would you mind deleting them?

Comment (kulisfunk) [reply client/src/models/index.js]: ok no problem

File: client/src/models/index.js (hunk 1 above)
Comment (kulisfunk) [reply client/src/models/index.js]: the file shouldn't have been there, I deleted it.

File: client/src/models/index.js (hunk 2 above)
Comment (kulisfunk) [reply client/src/models/index.js]: this is the same file Fiona picked up, it's deleted now

**Review by Findsay — APPROVED** [review]

**Pull request merged**